specification/common-types/resource-management/v5/networksecurityperimeter.json

```diff
@@ -0,0 +1,400 @@
+{
+  "swagger": "2.0",
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "info": {
+    "version": "2023-09-15-preview",
```

Comment:
What is the significance of the version here?  Don't we wnt this tobe generic (0000-00-00, or the like)?

Reply:
Yes, thanks - this should be generic, I was just trying to keep some swagger schema validation happy.

Reply:
just need to put in the blank version (0000-00-00)

Reply:
You can actually say zeroth day zeroth month? I assumed that would be invalid.